sql_generators/glean_usage/event_monitoring_live.py

```diff
@@ -35,6 +39,21 @@ def __init__(self) -> None:
         self.custom_render_kwargs = {}
         self.base_table_name = "events_v1"
 
+    def _get_tables_with_events(self, v1_name: str) -> Set[str]:
+        pings = set()
+        resp = requests.get(METRICS_INFO_URL.format(app_name=v1_name))
+        if resp.status_code == 404:
```

Comment:
Are there 404s when you run this?  I think an empty json should still exist for apps that don't have metrics

Reply:
I though I did during testing, but trying it out now all seems good. Removed that part